src/components/DiscoverModal/DiscoverModal.vue

```diff
@@ -24,12 +24,58 @@ export default {
   data() {
     return {
       validate: false,
-      favorite: false
+      favorite: false,
+      localUser: []
     }
   },
   props: {
     content: { type: Object, required: true },
     dialog: { type: Boolean, default: false }
+  },
+  created() {
+    this.localUser = this.getLocalUser()
```

Comment:
You can use a computed here 
`computed: {
  localUser() { return JSON.parse(localStorage.getItem('userLog"))}
}`